tests/ext/test_updater.py

```diff
@@ -679,9 +686,13 @@ async def delete_webhook(*args, **kwargs):
     @pytest.mark.parametrize("ext_bot", [True, False])
     @pytest.mark.parametrize("drop_pending_updates", [True, False])
     @pytest.mark.parametrize("secret_token", ["SecretToken", None])
+    @pytest.mark.parametrize("unix", [None, True])
     async def test_webhook_basic(
-        self, monkeypatch, updater, drop_pending_updates, ext_bot, secret_token
+        self, monkeypatch, updater, drop_pending_updates, ext_bot, secret_token, unix, file_path
     ):
+        # Skipping unix test on windows since they fail
+        if unix and platform.system() == "Windows":
+            pytest.skip("Windows doesn't support unix bind")
```

Comment:
why not make this a fixture like [here](https://github.com/python-telegram-bot/python-telegram-bot/blob/592c6cc6d349f3d1b84c270fb9efce283e0e7f28/tests/ext/test_ratelimiter.py#L144-L147)?

Reply:
because all other params should be tested on windows, and I would hate to copy paste the code just to have the skip as a decorator

Reply:
ah, fair enough :D